plugins/cuda/pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[build-system]
```

Comment:
What's the toml file used for?

Reply:
This is a replacement for `setup.py` https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html